tabcmd/commands/datasources_and_workbooks/delete_command.py

```diff
@@ -31,26 +32,42 @@ def run_command(args):
         logger.debug(_("tabcmd.launching"))
         session = Session()
         server = session.create_session(args)
+        content_type: str = ""
+        if args.workbook:
+            content_type = "workbook"
+        elif args.datasource:
+            content_type = "datasource"
 
-        logger.info(_("delete.status").format(args.name, ""))
+        container: TSC.ProjectItem = DeleteCommand.get_project_by_name_and_parent_path(
+            logger, server, args.project_name, args.parent_project_path
+        )
+        if container:
+            item_name = (args.parent_project_path or "") + "/" + (args.project_name or "default") + "/" + args.name
+        else:
+            Errors.exit_with_error(logger, "Containing project could not be found")
+        logger.info(_("delete.status").format(content_type, item_name or args.name))
 
         error = None
-        try:
-            item_to_delete = DeleteCommand.get_workbook_item(logger, server, args.name)
-            item_type = "workbook"
-        except TSC.ServerResponseError as workbook_error:
-            error = workbook_error
-        try:
-            item_to_delete = DeleteCommand.get_data_source_item(logger, server, args.name)
-            item_type = "datasource"
-        except TSC.ServerResponseError as ds_error:
-            error = ds_error
-        if not item_type:
+        if args.workbook or not content_type:
+            logger.debug("Attempt as workbook")
+            try:
+                item_to_delete = DeleteCommand.get_workbook_item(logger, server, args.name, container)
+                content_type = "workbook"
+            except TSC.ServerResponseError as error:
+                logger.debug(error)
+        if args.datasource or not content_type:
+            logger.debug("Attempt as datasource")
```

Comment:
localized?

Reply:
rolling over to https://github.com/tableau/tabcmd/issues/155